cmd/vic-machine/create/create.go

```diff
@@ -1610,6 +1610,11 @@ func splitVnetParam(p string) (vnet string, value string, err error) {
 		return
 	}
 
+	if strings.Contains(vnet, " ") && (len(mapped) == 1 || (len(mapped) == 2 && len(mapped[1]) == 0)) {
```

Comment:
I've got no idea why this test opens with a clause checking for a space in the vsphere network name - vsphere names can contain spaces and our behaviour should be consistent regardless. This needs changing or a comment adding as to what it's trying to accomplish.

If we wish to enforce that both parts of the network name are specified for container networks, then putting the error here https://github.com/morris-jason/vic/blame/69f836bcaaf0639b11382dcb116f5051ad09663b/cmd/vic-machine/create/create.go#L767 accomplishes that without changing behaviour for other options using this function.